.config/nextest.toml

```diff
@@ -0,0 +1,6 @@
+[profile.ci]
+retries = 3
```

Comment:
actually, i prefer 2, sorry. if a test fails twice, in a row, it is a big deal

Reply:
No problem.

Reply:
done